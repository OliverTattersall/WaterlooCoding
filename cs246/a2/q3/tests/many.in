a 0 b 1 1 4 4
a 0 b 1 0 3 4
a 0 b 0 1 3 4
pc0
a 0 b 10 0 8 1
pc0
p r 0 0
p r 0 1
p r 0 2
p r 0 3
