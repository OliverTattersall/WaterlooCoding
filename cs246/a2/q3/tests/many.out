Dimensions: 5x5
Number of rectangles: 3
	[Blue (1,1) 4x4]
	[Blue (1,0) 3x4]
	[Blue (0,1) 3x4]
Dimensions: 8x11
Number of rectangles: 4
	[Blue (1,1) 4x4]
	[Blue (1,0) 3x4]
	[Blue (0,1) 3x4]
	[Blue (10,0) 8x1]
[Blue (1,1) 4x4]
[Blue (1,0) 3x4]
[Blue (0,1) 3x4]
[Blue (10,0) 8x1]
